frontend/apps/reader/modules/readerview.lua

```diff
@@ -1050,9 +1050,9 @@ function ReaderView:checkAutoSaveSettings()
 
     local interval = G_reader_settings:readSetting("auto_save_settings_interval_minutes")
     interval = TimeVal:new{ sec = interval*60, usec = 0 }
-    local now_tv = UIManager:getTime()
-    if now_tv - self.settings_last_save_tv >= interval then
-        self.settings_last_save_tv = now_tv
+    local now_btv = UIManager:getTime() + Device.total_standby_tv
```

Comment:
Are we sure `UIManager:getTime()` always returns a "time since boot", on all platforms, where it makes sense to add this `total_standby_tv` (meaning it's some real clock time that disappeared from the "time since boot" accounting) ?

You would ask me, I would answer no - and these lines look (again) super weird to me.

Reply:
If it's the case, may be this `UIManager:getTime() + Device.total_standby_tv` should be hidden in a `UIManager:getElapsedTimeSinceBoot()`, so at least, it reads more naturally here and elsewhere.

Reply:
GetTime is always MONOTONIC, which is technically uptime since a specific but undefined point in time. It may or may not be boot. It may or may not tick during suspend. On Linux, it's always boot, and it never ticks during suspend.

Reply:
@NiLuJe  You have written "not tick during suspend". Does this also holds true with standby?

A question to the ticking: Is it safe to assume, that we standby only on Linux devices? (It is clear, that we don't standby on Android.) But do we support other (maybe ancient) devices? 

Well, we use sysfs for this, so it should be safe.

Reply:
@NiLuJe Would it do any harm, if we add the total_Standby_tv in UIManager:getTime?

```
function UIManager:getTime()
    return self._now + Device.total_standby_time
end
```

The results of getTime are still monotonic, but may have jumps forward.

Reply:
Yeah, I wouldn't touch getTime itself, and use a new method instead, as was proposed ;).

Reply:
> @NiLuJe  You have written "not tick during suspend". Does this also holds true with standby?
> 
> A question to the ticking: Is it safe to assume, that we standby only on Linux devices? (It is clear, that we don't standby on Android.) But do we support other (maybe ancient) devices? 
> 
> Well, we use sysfs for this, so it should be safe.

No ticking on standby either, yeah.

And, yeah, we only PM ourselves on Linux ;).